src/v0/index.js

```diff
@@ -152,8 +152,8 @@ export default () => {
         const slots = {};
         const recordSlots = node => (slots[node.getAttribute && node.getAttribute('slot') || '__default'] = true);
 
-        addedNodes.forEach(recordSlots);
-        removedNodes.forEach(recordSlots);
+        [...addedNodes].forEach(recordSlots);
```

Comment:
The only issue I see with this is that it can be expensive because this will do `Array.from()` in ES2015 natively and `Array.prototype.slice.call()` in ES5 (which is very expensive in performance critical code.

We might be able to get away with `Array.prototype.forEach.call(addedNodes)`. Failing that just putting an `if (addedNodesOrRemovedNOdes)` check around these would suffice.